odoo/fields.py

```diff
@@ -978,6 +978,8 @@ def determine_value(self, record):
                 self.compute_value(record)
             else:
                 recs = record._in_cache_without(self)
+                if len(recs) > PREFETCH_MAX:
```

Comment:
> Nice! So this would affect all computed fields, not just related/inherited fields, right?

Yeah. That may have been intentional (to allow bigger batches on compute fields) though.

> Shouldn't this be fixed as of 8.0, as it looks like the issue was already there?

Yeah it's from the beginning of the new API.